### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by marcelgerber**
@larz0 @ingorichter I'd like to get that in fast as the current translation is even shittier

### CODE REVIEW

**Comment by larz0** [comment]
I'm going to merge this now. (cc @ingorichter)

**Comment by pthiess** [comment]
Thanks @MarcelGerber and @larz0 -:)

**Comment by marcelgerber** [comment]
@pthiess or @ingorichter It'd be great to have a post-merge review

File: locales/de/translation.json [hunk 1]
```diff
@@ -65,18 +43,15 @@
                 "new-features": {
                     "new-features": "Neue Features",
                     "new-features-in": "Neu in __sprintNumber__",
-                    "release-notes": {
-                        "past": "Alte Versionshinweise durchlesen…",
-                        "more": "Mehr in den Versionhinweisen."
-                    }
+                    "release-notes": "Mehr in den Versionhinweisen."
```
Comment (ingorichter) [inline locales/de/translation.json:46]: sollte das nicht Versionshinweisen heissen?

**Pull request merged**